dev-lisp/clisp_modules/clisp_modules-2.49.93~git.recipe

```diff
@@ -0,0 +1,101 @@
+SUMMARY="ANSI Common Lisp Implementation"
+DESCRIPTION="GNU CLISP is an ANSI Common Lisp Implementation. \
+It is known for high portability, small image size and efficient bignums. \
+\
+This package contains selected modules used by CLISP."
+HOMEPAGE="https://clisp.sourceforge.io/"
+COPYRIGHT="1992-1993 Bruno Haible, Michael Stoll
+	1994-1997 Bruno Haible, Marcus Daniels
+	1998 Bruno Haible, Pierpaolo Bernardi, Sam Steingold
+	1999-2000 Bruno Haible, Sam Steingold
+	2001-2018 Sam Steingold, Bruno Haible
+	"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="386dcd71ada798f317366c3a775c0d47292f8d0b"
+SOURCE_URI="https://gitlab.com/gnu-clisp/clisp/-/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="ff0b5dd48162c9d58c0801c2afa600b37dcb0ca9c41289cef87a1f98d6029595"
+SOURCE_DIR="clisp-$srcGitRev"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	clisp_modues$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	clisp$secondaryArchSuffix
+	lib:db$secondaryArchSuffix
+	lib:gdbm$secondaryArchSuffix
+	lib:libldap$secondaryArchSuffix
+	lib:libpcre$secondaryArchSuffix
+	lib:libpq$secondaryArchSuffix
+	lib:libsvm$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+#	lib:pari$secondaryArchSuffix
+#	pari_data$secondaryArchSuffix
+
+# TODO How to specify PostgreSQL (libpq) version 11 or less?
+#	devel:pari$secondaryArchSuffix
+#	pari_data$secondaryArchSuffix
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libdb$secondaryArchSuffix
+	devel:libffcall$secondaryArchSuffix
+	devel:libgdbm$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:libldap$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
+	devel:libpcre$secondaryArchSuffix
+	devel:libpq$secondaryArchSuffix
+	devel:libreadline$secondaryArchSuffix >= 8
+	devel:libsigsegv$secondaryArchSuffix
+	devel:libsvm$secondaryArchSuffix
+	devel:libunistring$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:cmp
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:msgfmt
+	"
+
+BUILD()
+{
+	./configure \
+		FORCE_UNSAFE_CONFIGURE=1 \
+		CXXFLAGS="_D_BSD_SOURCE" \
+		LDFLAGS="-lbsd -lnetwork" \
+		--with-included-regex \
+		--prefix=$prefix \
+		--datarootdir=$dataDir \
+		--bindir=$prefix/bin \
+		--libdir=$libDir \
+		--docdir=$docDir \
+		--mandir=$manDir \
+		build-dir
+	#make
+}
+
+INSTALL()
+{
+	mkdir -p build-dir/clx/mit-clx/
+	cp modules/clx/mit-clx/README.CLISP build-dir/clx/mit-clx/
+	cd build-dir
+	make install-modules MODULES="berkeley-db"
+	make install-modules MODULES="clx/mit-clx"
+	make install-modules MODULES="editor"
+	make install-modules MODULES="gdbm"
+	make install-modules MODULES="libsvm"
+#	make install-modules MODULES="pari" MODULE_CONFIGURE_FLAGS=--with-libpary-prefix=/boot/system/data/ pari # configure: error: "Could not identify PARI data directory"
```

Comment:
Didn't get this pari thing working here also. So no worries for now. 👍

Reply:
Pari module needs `pari.desc` file from `pari_devel` package. It is not more packaged, because its contents can be recreated otherwise (it contains details about available Pari functions). It seems this file can be generated during building of `pari`, but I am not sure in which way. If I discover this way, first `sci-mathematics/pari` recipe needs to be updated, after that CLISP module `pari` can be built with updated `pari_devel` package.

Another possibility is for CLISP to not rely on `pari.desc` package, but instead to extract this extra-information by other means. Unfortunately, I do not see in `pari`, `pari_devel` or `pari_data` anything from where to take this information. Only file structure during building of `pari` seems to contain this information. If this approach can work, it will be fixed upstream in CLISP.

Reply:
If you could get it to work it's fine, if not, I'm not going to fight over this :D 
Busy atm with some other stuff taking my focus away, so I'll be a bit in the background for a while. :)

Reply:
Of course, I will take care of `pari` if I find a solution.